desktop/core/src/desktop/api2.py

```diff
@@ -46,8 +46,8 @@
 from beeswax.models import Namespace
 from desktop import appmanager
 from desktop.auth.backend import is_admin
-from desktop.conf import ENABLE_CONNECTORS, ENABLE_GIST_PREVIEW, CUSTOM, get_clusters, ENABLE_SHARING
-from desktop.conf import ENABLE_NEW_STORAGE_BROWSER
+from desktop.conf import ENABLE_CONNECTORS, ENABLE_GIST_PREVIEW, CUSTOM, get_clusters, IS_K8S_ONLY, ENABLE_SHARING
```

Comment:
Not used in this file?

Reply:
didnt change anything in this line. Not sure why this is showing up as a change.

Reply:
Try rebasing your branch with latest master changes once then.

Reply:
This is still present